packages/react/src/components/TeachingBubble/TeachingBubble.styles.ts

```diff
@@ -223,12 +228,16 @@ export const getStyles = (props: ITeachingBubbleStyleProps): ITeachingBubbleStyl
           [`.${classNames.buttonLabel}`]: fonts.medium,
           ':hover': {
             backgroundColor: palette.themeLighter,
-            borderColor: palette.themeLighter,
             color: palette.themePrimary,
+            border: `2px solid ${palette.black}`,
+            outline: `1px solid ${palette.white}`,
+            outlineOffset: '-3px',
```

Comment:
We don't need these styles for hover, only focus

Reply:
I've reverted the `:hover` style back to how it was originally